visualization/app/codeCharta/ui/metricChooser/metricChooser.component.scss

```diff
@@ -1,10 +1,28 @@
 $optionHeight: 48;
 
+cc-metric-value-hovered,
 area-metric-chooser-component,
 height-metric-chooser-component,
 color-metric-chooser-component,
 distribution-metric-chooser-component,
 edge-chooser-component {
+	.metric-value {
+		height: 25px;
+		float: right;
+		position: absolute;
+		right: 0;
+		margin-top: -9px;
+
+		.rounded-box {
+			margin-top: -2px;
+			margin-left: -2px;
```

Comment:
Hacky sollution, i suggest removing both of those margins and aligning the whole container by making the
"cc-metric-value-hovered"
`position: absolute;
right: 2px;
top: 2px`